macos/snd_.ksy

```diff
@@ -0,0 +1,201 @@
+meta:
+  id: snd_
+  endian: be
+  xref:
+    wikidata: Q7564684
+    reference: "https://developer.apple.com/library/archive/documentation/mac/pdf/Sound/Sound_Manager.pdf"
```

Comment:
This should be specified in the [`doc-ref` key](https://doc.kaitai.io/user_guide.html#doc-ref) instead.

```suggestion
doc-ref: "https://developer.apple.com/library/archive/documentation/mac/pdf/Sound/Sound_Manager.pdf"
```

Reply:
Changed to doc-ref